electron/types.ts

```diff
@@ -258,6 +258,7 @@ export interface InstallParams {
   sdlList?: Array<string>
   installLanguage?: string
   runner: Runner
+  platformToInstall: 'Windows' | 'Mac' | 'Linux'
```

Comment:
You calling everywhere toLowerCase. 
Why not declare everything here in lower case and get rid of the toLowerCase calls.

Maybe i missed something, why you doing this ?

Reply:
I will check if legendary accepts it with all lower case, because on their docs says that it is CamelCase.
So then I start from this and then need to lowercase because this is how gogdl works.

Reply:
Yes, checking here and Legendary only accepts Camel case:
``` ./public/bin/linux/legendary install 19927295d6e3467887d4e830d8c85963 --platform mac
[cli] WARNING: Platform "mac" may be invalid. Valid ones are: Windows, Win32, Mac.
```